manifests/c/ConselhoNacionaldeJustica-CNJ/pje-office/1.0.28/ConselhoNacionaldeJustica-CNJ.pje-office.locale.pt-BR.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ConselhoNacionaldeJustica-CNJ.pje-office
+PackageVersion: 1.0.28
+PackageLocale: pt-BR
+Publisher: Conselho Nacional de Justica - CNJ
+PackageName: pje-office
+License: GNU GENERAL PUBLIC LICENSE
+ShortDescription: The PJeOffice is a software provided by the CNJ for electronic signature of documents for the PJe system.

```

```suggestion
ShortDescription: O PJeOffice é um software disponibilizado pelo CNJ para assinatura eletrônica de documentos do sistema PJe.
```
(Internet translation - feel free to improve.)

The short description should be in the localization file's language.